### PR TITLE
Update/feed component from api

### PR DESCRIPTION
**Pull request by yohanlebret**

### CODE REVIEW

File: src/entrypoints/admin/comments.js [hunk 1]
```diff
@@ -247,33 +247,40 @@ window.comments = (() => {
...
     }
+  }
 
-    const updateCommentVisibility = (visible) => {
-      // Show/hide comments
-      commentApp.setVisible(visible);
+  function initCommentsInterfaceFromApi(commentsElement, commentsOutputElement, apiUrl, apiCommentsEntrypoint,
+    mode = 'cors', headerOptions = {}, componentStyle = null) {
+    const headers = new Headers(headerOptions);
+    const resquestOptions = {
```
Comment (ColinBeeby-Developer) [inline src/entrypoints/admin/comments.js:268]: resquestOptions - is this a typo for requestOptions ? Correct for readability?

File: src/entrypoints/admin/comments.js [hunk 2]
```diff
@@ -247,33 +247,40 @@ window.comments = (() => {
...
+      method: 'POST',
+      headers,
+      mode,
     };
+    const request = new Request(
+      apiUrl + apiCommentsEntrypoint,
+      resquestOptions,
```
Comment (ColinBeeby-Developer) [inline src/entrypoints/admin/comments.js:275]: resquestOptions ?

File: src/entrypoints/admin/comments.js (hunk 1 above)
Comment (yohanlebret) [reply src/entrypoints/admin/comments.js]: Rename `resquestOptions` with `requestOptions`

File: src/entrypoints/admin/comments.js (hunk 2 above)
Comment (yohanlebret) [reply src/entrypoints/admin/comments.js]: Rename `resquestOptions` with `requestOptions`

**Review by the-nathan-smith — COMMENTED** [review]
Looks good to me.

**Review by ColinBeeby-Developer — APPROVED** [review]
All looks good to me.

**Pull request merged**